docs/api-reference/cli.md

```diff
@@ -106,6 +106,22 @@ PORT=4000 npx next start
 
 > Note: `PORT` can not be set in `.env` as booting up the HTTP server happens before any other code is initialized.
 
+### Working with Load-balancer Timeouts
+
+When deploying Next.js behind a downstream proxy (e.g. a load-balancer like AWS ELB/ALB) it's important to configure Next's underlying http-server with [keep-alive timeouts](https://nodejs.org/api/http.html#http_server_keepalivetimeout) that are _larger_ than the downstream proxies' timeouts. Otherwise, once a keep-alive timeout is reached for a given TCP connection, Node.js will immediately terminate that connection without notifying the downstream proxies. This results in the downstream proxies throwing intermittent 5xx errors whenever they attempt to reuse a connection that Node.js has already terminated.
+
+To configure the timeout values for the production Next.js server, pass `--keepAliveTimeout` (in milliseconds) to `next start`, like so:
+
+```bash
+npx next start --keepAliveTimeout 70000
+```
+
+In some earlier versions of Node.js, there is [a regression that requires that `headersTimeout` is set to a value greater than the `keepAliveTimeout` value](https://github.com/nodejs/node/issues/27363). To configure [`headersTimeout`](https://nodejs.org/api/http.html#serverheaderstimeout) for the production Next.js server, pass `--headersTimeout` (in milliseconds) to `next start`, like so:
```

Comment:
> In some earlier versions of Node.js

In which earlier versions? It looks like Node.js 10 and earlier but Next.js only supports 12+ (soon to be 14+) so I think we don't need this setting.

Reply:
The [fix doesn't appear until 12.19](https://github.com/nodejs/node/issues/27363#issuecomment-705085158) it looks like. Let's keep this setting until Next.js drops support for 12?

Reply:
Node.js 12 reaches EOL April 30th. We might not even have a stable release of Next.js before then. I don't think the additional flags and docs are worth it.

Reply:
headersTimeout is removed